Formula/mlpack.rb

```diff
@@ -0,0 +1,93 @@
+class Mlpack < Formula
+  desc "Scalable C++ machine learning library"
+  homepage "https://www.mlpack.org"
+  url "https://mlpack.org/files/mlpack-3.2.2.tar.gz"
+  sha256 "7aef8c27645c9358262fec9ebba380720a086789d6519d5d1034346412a52ad6"
+
+  depends_on "cmake" => :build
+  depends_on "doxygen" => :build
+  depends_on "pkg-config" => :build
+  depends_on "armadillo"
+  depends_on "boost"
+  depends_on "ensmallen"
+  depends_on "graphviz"
+
+  def install
+    # stb
+    resource "stb" do
+      url "https://github.com/nothings/stb/archive/f67165c2bb2af3060ecae7d20d6f731173485ad0.tar.gz"
+      sha256 "ad5d34b385494cf68c52fad5762d00181c0c6d4787988fc75f17295c3c726bf8"
+    end
+    resource("stb").stage do
+      (include/"stb").install "stb.h"
+      (include/"stb").install "stb_easy_font.h"
+      (include/"stb").install "stb_leakcheck.h"
+      (include/"stb").install "stb_truetype.h"
+      (include/"stb").install "stb_c_lexer.h"
+      (include/"stb").install "stb_herringbone_wang_tile.h"
+      (include/"stb").install "stb_perlin.h"
+      (include/"stb").install "stb_voxel_render.h"
+      (include/"stb").install "stb_connected_components.h"
+      (include/"stb").install "stb_image.h"
```

Comment:
@rcurtin I've added every stb header into this directory as a resource.

Reply:
:+1: that's probably overkill, we only use `stb_image.h` and `stb_image_write.h`, but I suppose it can't hurt.

Reply:
- Are the headers used by `mlpack` at runtime? If not, put them inside `buildpath`.
- Can we install all headers with globbing? `destdir.install Dir["*.h"]`

Reply:
If a user wants to use mlpack for image-related tasks, the headers are used at runtime.